src/build_settings.cpp

```diff
@@ -557,6 +559,14 @@ gb_global TargetMetrics target_openbsd_amd64 = {
 	str_lit("e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"),
 };
 
+gb_global TargetMetrics target_netbsd_amd64 = {
+	TargetOs_netbsd,
+	TargetArch_amd64,
+	8, 8, 8, 16,
+	str_lit("x86_64-unknown-netbsd-elf"),
+	str_lit("e-m:w-i64:64-f80:128-n8:16:32:64-S128"),
```

Comment:
Also with LLVM 18, this line can be removed

Reply:
Got it.

But how should this be handled when building with LLVM17? I did not see any branches on LLVM version.
It will take a while before 18 lands in pkgsrc. 17 just landed in current and there wont be binaries until Q2 2024.

Reply:
Ah I was not clear, the llvm 18 branch that was merged removed this field from the struct completely, for all versions of llvm. This field was actually dead code all along, it didn't do anything.

Reply:
Yea.. I just figured it out. :D
Sorry. I missed that 18 was merged.